python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -159,6 +159,21 @@ class FgcmBuildStarsConfig(pexConfig.Config):
         dtype=str,
         default="calib_psf_candidate"
     )
+    doSubtractLocalBackground = pexConfig.Field(
+        doc="Subtract the local background before performing calibration?",
+        dtype=bool,
+        default=False
+    )
+    calibFluxApertureRadius = pexConfig.Field(
+        doc="Calibration flux aperture radius (pixels)",
+        dtype=float,
+        default=12.0
```

Comment:
There seems to be some danger for things getting out of sync here.  The flux to be corrected is set in the config `instFluxField` which defaults to `slot_CalibFlux_instFlux` (I realize the default is `base_CircularApertureFlux_12_0_instFlux` but, in practice, this could be set to anything...possibly not even circular aperture at all!)  I believe the radius here should match that of `instFluxField`, but you have no guarantee that this will be `base_CircularApertureFlux_12_0_instFlux`.  This radius should probably be gleaned in the code based on the particular flux type stored in the `instFluxField` config.